src/clib/pio.h

```diff
@@ -107,6 +245,9 @@ typedef struct var_desc_t
 {
     /* Variable ID. */
     int varid;
+
+    /* Variable name - cached */
+    char vname[PIO_MAX_NAME];
```

Comment:
Should we use [PIO_MAX_NAME + 1] here? In line 760 of pioc_nc.c "char my_name[PIO_MAX_NAME + 1];" is used.

Reply:
I somehow don't like the PIO_MAX_NAME+1 style (add 1 for the string delimiter). It shouldn't be an issue unless we have a very long variable name and strncpy() does not add the delimiter. In any case we won't leak.